pom.xml

```diff
@@ -17,7 +17,7 @@
   <url>http://activiti.org</url>
   <properties>
     <activiti-build.version>7.0.39</activiti-build.version>
-    <activiti-api.version>7.0.37</activiti-api.version>
+    <activiti-api.version>7.0.0-SNAPSHOT</activiti-api.version>
```

Comment:
@ryandawsonuk ? do we need this? I think that we should keep the fixed version here.